[2024-06-05T02:18:42.099+0000] {processor.py:161} INFO - Started process (PID=14936) to work on /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:18:42.100+0000] {processor.py:830} INFO - Processing file /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-05T02:18:42.101+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:18:42.101+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:18:42.105+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:18:42.104+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-05T02:18:42.105+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:18:42.105+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-05T02:18:42.105+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:18:42.117+0000] {processor.py:183} INFO - Processing /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2024-06-05T02:19:34.079+0000] {processor.py:161} INFO - Started process (PID=14987) to work on /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:19:34.081+0000] {processor.py:830} INFO - Processing file /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-05T02:19:34.083+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:19:34.082+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:19:34.086+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:19:34.085+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-05T02:19:34.086+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:19:34.086+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-05T02:19:34.086+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:19:34.097+0000] {processor.py:183} INFO - Processing /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2024-06-05T02:20:23.959+0000] {processor.py:161} INFO - Started process (PID=15044) to work on /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:20:23.960+0000] {processor.py:830} INFO - Processing file /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-05T02:20:23.962+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:20:23.961+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:20:23.965+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:20:23.964+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-05T02:20:23.966+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:20:23.965+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-05T02:20:23.966+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:20:23.977+0000] {processor.py:183} INFO - Processing /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2024-06-05T02:21:24.513+0000] {processor.py:161} INFO - Started process (PID=15107) to work on /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:21:24.514+0000] {processor.py:830} INFO - Processing file /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-05T02:21:24.516+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:21:24.516+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:21:24.519+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:21:24.519+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-05T02:21:24.520+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:21:24.519+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-05T02:21:24.520+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:21:24.530+0000] {processor.py:183} INFO - Processing /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2024-06-05T02:22:19.547+0000] {processor.py:161} INFO - Started process (PID=15165) to work on /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:22:19.548+0000] {processor.py:830} INFO - Processing file /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-05T02:22:19.550+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:22:19.550+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:22:19.553+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:22:19.553+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-05T02:22:19.554+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:22:19.554+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-05T02:22:19.554+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:22:19.565+0000] {processor.py:183} INFO - Processing /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2024-06-05T02:23:16.759+0000] {processor.py:161} INFO - Started process (PID=15225) to work on /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:23:16.760+0000] {processor.py:830} INFO - Processing file /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-05T02:23:16.762+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:23:16.762+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:23:16.766+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:23:16.765+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-05T02:23:16.766+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:23:16.766+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-05T02:23:16.766+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:23:16.777+0000] {processor.py:183} INFO - Processing /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2024-06-05T02:25:46.647+0000] {processor.py:161} INFO - Started process (PID=15354) to work on /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:25:46.649+0000] {processor.py:830} INFO - Processing file /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-05T02:25:46.651+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:25:46.650+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:25:46.654+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:25:46.653+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-05T02:25:46.654+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:25:46.654+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-05T02:25:46.655+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:25:46.666+0000] {processor.py:183} INFO - Processing /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2024-06-05T02:26:40.070+0000] {processor.py:161} INFO - Started process (PID=15405) to work on /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:26:40.071+0000] {processor.py:830} INFO - Processing file /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-05T02:26:40.073+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:26:40.072+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:26:40.076+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:26:40.076+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-05T02:26:40.077+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:26:40.077+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-05T02:26:40.077+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:26:40.088+0000] {processor.py:183} INFO - Processing /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2024-06-05T02:27:28.795+0000] {processor.py:161} INFO - Started process (PID=15456) to work on /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:27:28.797+0000] {processor.py:830} INFO - Processing file /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-05T02:27:28.798+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:27:28.798+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:27:28.802+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:27:28.801+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-05T02:27:28.802+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:27:28.802+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-05T02:27:28.803+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:27:28.814+0000] {processor.py:183} INFO - Processing /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2024-06-05T02:28:28.286+0000] {processor.py:161} INFO - Started process (PID=15517) to work on /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:28:28.288+0000] {processor.py:830} INFO - Processing file /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-05T02:28:28.290+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:28:28.289+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:28:28.293+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:28:28.292+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-05T02:28:28.293+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:28:28.293+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-05T02:28:28.294+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:28:28.304+0000] {processor.py:183} INFO - Processing /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2024-06-05T02:29:30.988+0000] {processor.py:161} INFO - Started process (PID=15585) to work on /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:29:30.990+0000] {processor.py:830} INFO - Processing file /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-05T02:29:30.991+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:29:30.991+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:29:30.995+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:29:30.995+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-05T02:29:30.996+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:29:30.995+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-05T02:29:30.996+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:29:31.007+0000] {processor.py:183} INFO - Processing /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2024-06-05T02:30:33.947+0000] {processor.py:161} INFO - Started process (PID=15653) to work on /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:30:33.952+0000] {processor.py:830} INFO - Processing file /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-05T02:30:33.953+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:30:33.953+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:30:33.957+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:30:33.956+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-05T02:30:33.957+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:30:33.957+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-05T02:30:33.957+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:30:33.968+0000] {processor.py:183} INFO - Processing /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2024-06-05T02:31:38.501+0000] {processor.py:161} INFO - Started process (PID=15718) to work on /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:31:38.503+0000] {processor.py:830} INFO - Processing file /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-05T02:31:38.505+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:31:38.505+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:31:38.509+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:31:38.508+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-05T02:31:38.509+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:31:38.509+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-05T02:31:38.509+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:31:38.520+0000] {processor.py:183} INFO - Processing /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2024-06-05T02:32:41.021+0000] {processor.py:161} INFO - Started process (PID=15783) to work on /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:32:41.022+0000] {processor.py:830} INFO - Processing file /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-05T02:32:41.024+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:32:41.024+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:32:41.028+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:32:41.027+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-05T02:32:41.028+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:32:41.028+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-05T02:32:41.028+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:32:41.039+0000] {processor.py:183} INFO - Processing /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2024-06-05T02:33:46.230+0000] {processor.py:161} INFO - Started process (PID=15847) to work on /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:33:46.232+0000] {processor.py:830} INFO - Processing file /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-05T02:33:46.233+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:33:46.233+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:33:46.237+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:33:46.236+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-05T02:33:46.237+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:33:46.237+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-05T02:33:46.237+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:33:46.248+0000] {processor.py:183} INFO - Processing /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2024-06-05T02:34:45.107+0000] {processor.py:161} INFO - Started process (PID=15909) to work on /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:34:45.108+0000] {processor.py:830} INFO - Processing file /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-06-05T02:34:45.110+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:34:45.110+0000] {dagbag.py:545} INFO - Filling up the DagBag from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:34:45.114+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:34:45.113+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-06-05T02:34:45.114+0000] {logging_mixin.py:188} INFO - [2024-06-05T02:34:45.114+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-06-05T02:34:45.114+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-06-05T02:34:45.125+0000] {processor.py:183} INFO - Processing /home/ubuntu/endtoend_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
